cmd2/cmd2.py

```diff
@@ -3358,7 +3358,7 @@ def do_history(self, args: argparse.Namespace) -> Optional[bool]:
             with os.fdopen(fd, 'w') as fobj:
                 for command in history:
                     if command.statement.multiline_command:
-                        fobj.write('{}\n'.format(command.expanded.rstrip()))
+                        fobj.write('{}\n'.format(command.expanded))
```

Comment:
Are these no longer needed because expanded already does an rstrip() internally?

Reply:
Aside from `Statement.raw`, its other members are built using tokens produced by shlex. Therefore all leading/trailing whitespace is gone.

The code changed in this Pull Request is referencing `HistoryItem.expanded` which just returns `Statement.expanded_command_line`. So no whitespace needs to be stripped.